custom/aaa/views.py

```diff
@@ -342,75 +359,92 @@ def post(self, request, *args, **kwargs):
         data = {}
 
         if sub_section == 'person_details':
-            person = dict(
-                name='Reena Kumar',
-                gender='Female',
-                status='Pregnant Woman',
-                dob=date(1991, 5, 11),
-                marriedAt=25,
-                aadhaarNo='Yes'
+            person_model = Woman if section != 'child' else Child
+
+            extra_select = {
+                'gender': 'sex',
+                'aadhaarNo': 'has_aadhar_number',
+                'address': 'hh_address',
+                'phone': 'contact_phone_number',
+                'religion': 'hh_religion',
+                'caste': 'hh_caste',
+                'bplOrApl': 'hh_bpl_apl',
+                'subcentre': '\'Rasidpur\'',
+                'village': '\'Rasidpur\'',
+                'anganwadiCentre': '\'Aspataal Ward\''
+            }
+
+            if section != 'child':
+                extra_select.update({
+                    'status': 'migration_status',
+                    'marriedAt': 'age_marriage'
+                })
+
+            extra_select_values = list(extra_select.keys())
+            extra_select_values.extend(
+                ['dob', 'name', 'husband_name']
             )
-            other = dict(
-                address='J-142, Saket, New Delhi, Delhi',
-                subcentre='Rasidpur',
-                village='Rasidpur',
-                anganwadiCentre='Aspataal Ward',
-                phone='844-860-4774',
-                religion='Hinduk',
-                caste='Sudra',
-                bplOrApl='BPL',
-
+            person = person_model.objects.extra(
+                select=extra_select
+            ).values(*extra_select_values).get(
+                domain=request.domain,
+                person_case_id=beneficiary_id
             )
+
             data = dict(
                 person=person,
-                other=other,
             )
 
             if section == 'child':
-                mother = dict(
-                    id=1,
-                    name='Reena Kumar',
-                    gender='Female',
-                    status='Pregnant Woman',
-                    dob=date(1991, 5, 11),
-                    marriedAt=25,
-                    aadhaarNo='Yes'
-                )
+                mother = Woman.objects.extra(
+                    select={
+                        'id': 'person_case_id'
+                    }
+                ).values('id', 'name').get(person_case_id=beneficiary_id)
                 data.update(dict(mother=mother))
             else:
+                # TODO update when the model will be created
                 husband = dict(
-                    name='Raju Kumar',
+                    name=person['husband_name'],
                     gender='Female',
```

Comment:
Can you replace these static values with 'N/A' (here and in general, so that its easier to find what's not yet done